code/modules/admin/topic.dm

```diff
@@ -9,7 +9,7 @@
 
 /datum/admins/proc/formatJobGroup(var/mob/mob, var/title, var/color, var/bantype, var/list/joblist)
 	. += "<tr bgcolor='[color]'><th><a href='?src=\ref[src];jobban3=[bantype];jobban4=\ref[mob]'>[title]</a></th></tr><tr><td class='jobs'>"
-	for(var/jobPos in GLOB.joblist)
+	for(var/jobPos in joblist)
 		. += formatJob(mob, jobPos, GLOB.joblist[jobPos])
```

Comment:
You sure this will work, it still takes `GLOB.joblist[jobPos]`?

Reply:
It will.
![image](https://cdn.discordapp.com/attachments/265411548963405824/892122799408283648/unknown.png)